### PR TITLE
 all the cert rotation controllers to be executed in run-once mode

### PR DESCRIPTION
**Pull request by deads2k**
this is needed for cert recovery.

### CODE REVIEW

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 1]
```diff
@@ -468,6 +468,26 @@ func (c *CertRotationController) Run(workers int, stopCh <-chan struct{}) {
...
 
+}
+
+// RunOnce will run the cert rotation logic, but will not try to update the static pod status.
+// This eliminates the need to pass an OperatorClient and avoids dubious writes and status.
+func (c *CertRotationController) RunOnce() error {
+	errlist := []error{}
+	for _, certRotator := range c.certRotators {
+		if err := certRotator.RunOnce(); err != nil {
+			errlist = append(errlist)
```
Comment (sttts) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]: append something

**Comment by tnozicka** [comment]
/hold
(missed the append)

**Comment by deads2k** [comment]
merged in https://github.com/openshift/cluster-kube-apiserver-operator/pull/461

**Pull request closed without merging**